### ISSUE TITLE
param `nrow` in `make_grid` is misleading

### ISSUE THREAD

**Issue by alykhantejani** [pytorch/vision#249]
```Python
import torch
import torchvision
label_img = torch.rand(100, 3, 32, 32)
x = torchvision.utils.make_grid(label_img, nrow=20, padding=10)
torchvision.transforms.ToPILImage()(x).save("example.png")
``` 

produces this image:
![example](https://user-images.githubusercontent.com/687194/30210372-4e31d660-9494-11e7-9736-3292952a6db7.png)

which has 5 rows and 20 columns. Although the [docs](http://pytorch.org/docs/master/torchvision/utils.html#torchvision.utils.make_grid) say 
```
nrow (int, optional) – Number of rows in grid. Final grid size is (B / nrow, nrow). Default is 8.
``` 

Which says the output will actually be `(B/nrow, nrow)`. Perhaps this param should be called `ncol`, or we should change the output shape to be `(nrow, B/nrow)`? 

I think, to keep backward compatibility changing the output shape to `(nrow, B/nrow)` would make more sense. 

Happy to send a PR if you agree?

**Comment by fmassa** [comment]
Maybe we could change the docs to read "number of images in a row"? This way we don't change the API, and it still makes sense from the parameter name?

**Comment by alykhantejani** [comment]
Yeah - this is a lot easier and wont have "unexpected" changes for people who are used to using it in its current form. I'll send a PR for this.

**Comment by fmassa** [comment]
Fixed via #250
